test/fuzz_group.cpp

```diff
@@ -831,7 +831,7 @@ void parse_and_apply_instructions(std::string& in, const std::string& path, util
                     if (log) {
                         *log << "sg_r.open(path);\n";
```

Comment:
The log outputs should also change to match (here and lines 836, and 847).
`*log << "sg_r.open(path, true, SharedGroupOptions(key);\n";`
The variable `key` is declared in the log at the top when opening the file for the first time.